draft-ietf-quic-qlog-main-schema.md

```diff
@@ -1137,18 +1143,76 @@ SimulationMarker = {
 ~~~
 {: #simulation-marker-def title="SimulationMarker definition"}
 
-# Event definition guidelines
+# Event extensibility {#event-extensibility}
+
+The main qlog schema defined by this document describes log and trace formats
+that have related events. A minimal set of common events are presented in
+{{events}}. It is expected that logging is extended with specific, per-protocol
+event definitions that specify the name (category + type) and data needed for
+each individual event. Examples include the QUIC event definitions {{QLOG-QUIC}}
+and HTTP/3 event definitions {{QLOG-H3}}.
+
+New event definitions SHOULD follow the guidance in this section, which provides
+consistency that makes it easier for qlog implementers to support a wide variety
+of protocols.
+
+## Extended Event Schema
+
+New event definitions SHOULD be part of an extension schema, using the
+annotations and concepts presented in this section.
+
+An extension schema defines one or more qlog event categories that can contain
+multiple event definitions. Each category MUST have a globally unique category
```

Comment:
we are being ambiguous as to what exactly an "event definition" is.

Above, you define it as `per-protocol
event definitions that specify the name (category + type) and data` while here, it's more correct to say each category contains multiple event types... in the next sentence then, you switch to just the word "event".

It's not a major issue, but I feel we need consistent naming for these things...

For that, I would say `event type` here instead of `event definition` and in the next sentence `Each event type MUST only belong` instead of `Each event MUST`


Related: I think it would really help to have an illustrative example of this at the start of this part